docs/source/getting_started/content/con_understanding-codeready-containers-configuration.adoc

```diff
@@ -0,0 +1,10 @@
+[id="understanding-codeready-containers-configuration_{context}"]
+= Understanding {prod} configuration
+
+Both the [command]`{bin}` binary and the {prod} virtual machine can be configured using the [command]`{bin} config` command.
+The [command]`{bin} config` command requires a subcommand to act on the configuration.
+The available subcommands are `get`, `set,` `unset`, and `view`.
+The `get`, `set`, and `unset` subcommands operate using named configurable properties.
+
+Preflight checks for the [command]`{bin} start` and [command]`{bin} setup` commands can be configured to be skipped or to report a warning rather than an error.
```

Comment:
I don't know what "preflight" means in this context. Is "preflight" a concept that you've already explained somewhere previously? If not, it sounds like jargon or slang to me, especially for translators.

"can be configured" is passive voice. I suggest rephrasing as I suggested in my previous similar comment.

Reply:
Good call on "preflight" being jargon. This is a term that was used previously in Minishift for checks which are run prior to starting the VM which provisions the OpenShift cluster.

Perhaps "startup checks" or "pre-start checks" would be more understandable. What do you think?

Reply:
the term pre-flight checks is mostly the internal naming and how we talk about them. Startup checks are more correct in term to the user.

Reply:
Is a startup check a check that takes place when something is started? I don't see the full context here, so I'm guessing you're talking about a virtual machine starting up (i.e. booting). Is that correct?

Anyway, "startup check" is more intuitive and less jargony to my ear.